engine/classes/Elgg/Values.php

```diff
@@ -173,4 +173,23 @@ public static function normalizeGuids(...$args) {
 	public static function preventViewOutput() {
 		return ['__view_output' => ''];
 	}
+	
+	/**
+	 * Check if a value isn't empty, but allow 0 and '0'
+	 *
+	 * @param mixed $value the value to check
+	 *
+	 * @see empty()
+	 *
+	 * @return bool
+	 * @since 3.0.0
+	 */
+	public static function isEmpty($value) {
```

Comment:
do we need $value to be defaulted to null to prevent php notices?

Reply:
nope, you could only get a php notice if you call it like `elgg_is_empty()` or `E\gg\Values::isEmpty()`, than you're just doing it wrong ;)